src/Data/Swagger/Internal.hs

```diff
@@ -334,10 +339,11 @@ data SwaggerItems t where
   SwaggerItemsPrimitive :: Maybe (CollectionFormat t) -> ParamSchema t -> SwaggerItems t
   SwaggerItemsObject    :: Referenced Schema   -> SwaggerItems Schema
   SwaggerItemsArray     :: [Referenced Schema] -> SwaggerItems Schema
+  deriving (Typeable)
 
 deriving instance Eq (SwaggerItems t)
 deriving instance Show (SwaggerItems t)
-deriving instance Typeable (SwaggerItems t)
+--deriving instance Typeable (SwaggerItems t)
```

Comment:
Should this be removed? Should `Eq` and `Show` be moved into the same `deriving` clause with `Typeable`?